test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
If any of these files were copied from bincrafters/conan-templates (as you indicated this one was), the copyright needs to be retained because they're MIT licensed.